### PR TITLE
aws.vpc | set-flow-log | Enables custom aggregation interval

### LINKED ISSUE #5454: aws.vpc | action: set-flow-log doesn't allow the user to set a custom MaxAggregationInterval
**Is your feature request related to a problem? Please describe.**
 Our use case is to set vpc flow logs aggregation to something less than 600 seconds. https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#EC2.Client.create_flow_logs
As of now it's not customizable and takes a default value of 600 seconds.

**Describe the solution you'd like**
Create a new param in `action: set-flow-log` called `MaxAggregationInterval` which would allow policy authors to use it. The default value would be: 600 seconds

### PR DESCRIPTION
**Pull request by PratMis**
Closes #5454

### CODE REVIEW

File: tests/test_vpc.py [hunk 1]
```diff
@@ -2689,3 +2689,34 @@ def test_vpc_delete_flow_logs(self):
...
         self.assertFalse(logs)
+
+    def test_vpc_set_flow_logs_maxaggrinterval(self):
+        session_factory = self.replay_flight_data("test_vpc_set_flow_logs_maxaggrinterval")
+        p = self.load_policy(
+            {
+                "name": "c7n-vpc-flow-logs-maxinterval",
+                "resource": "vpc",
+                "filters": [
+                    {"tag:Name": "FlowLogTest"}
```
Comment (kapilt) [inline tests/test_vpc.py]: should have a flow log filter here to verify pre-conditions.

**Comment by kapilt** [comment]
please have a look at running the script in #5452 to minimize extraneous data

File: tests/test_vpc.py (hunk 1 above)
Comment (PratMis) [reply tests/test_vpc.py]: Addressed, thanks!

**Comment by PratMis** [comment]
Used the script to minimize extraneous data! Thanks!!

**Pull request merged**